src/libs/PersonalDetailsUtils.ts

```diff
@@ -24,10 +25,28 @@ Onyx.connect({
     },
 });
 
+/**
```

Comment:
can you please reserve `/** */` JSDoc-style comments for:

- function signatures
- propType declarations (needed here for use in storybook)

Otherwise, stick to `//` for one-or-two line comments like this one, or `/* */` for larger block comments that aren't JSDocs?